test/JDBC/expected/babel_726-vu-verify.out

```diff
@@ -0,0 +1,575 @@
+12
+~~END~~
+
+
+-- int and smallint
+select coalesce(a14, a15) from babel_726_t1
+GO
+~~START~~
+int
+12
+~~END~~
+
+select coalesce(a15, a14) from babel_726_t1
+GO
+~~START~~
+int
+10
+~~END~~
+
+
+-- smallint and tinyint
+select coalesce(a15, a16) from babel_726_t1
+GO
+~~START~~
+smallint
+10
+~~END~~
+
+select coalesce(a16, a15) from babel_726_t1
+GO
+~~START~~
+smallint
+8
+~~END~~
+
+
+-- tinyint and bit
+select coalesce(a16, a17) from babel_726_t1
+GO
+~~START~~
+tinyint
+8
+~~END~~
+
+select coalesce(a17, a16) from babel_726_t1
+GO
+~~START~~
+tinyint
+1
+~~END~~
+
+
+-- bit and ntext. Throws error as CASTing from ntext to bit is not supported
+select coalesce(a17, a18) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: COALESCE could not convert type ntext to "bit")~~
+
+select coalesce(a18, a17) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: COALESCE could not convert type ntext to "bit")~~
+
+
+-- ntext and text
+select coalesce(a18, a19) from babel_726_t1
+GO
+~~START~~
+ntext
+asdf
+~~END~~
+
+select coalesce(a19, a18) from babel_726_t1
+GO
+~~START~~
+ntext
+asdfghjk
+~~END~~
+
+
+-- text and image.
+select coalesce(a19, a20) from babel_726_t1
+GO
+~~START~~
+text
+asdfghjk
+~~END~~
+
+select coalesce(a20, a19) from babel_726_t1
+GO
+~~START~~
+text
+0x31323334
+~~END~~
+
+
+-- image and uniqueidentifier. 
+select coalesce(a20, a21) from babel_726_t1
+GO
+~~START~~
+image
+31323334
+~~END~~
+
+select coalesce(a21, a20) from babel_726_t1
+GO
+~~START~~
+image
+A678F151C7532C479BE11C08942342D7
+~~END~~
+
+
+-- uniqueidentifier and nvarchar
+select coalesce(a21, a22) from babel_726_t1
+GO
+~~START~~
+uniqueidentifier
+51F178A6-53C7-472C-9BE1-1C08942342D7
+~~END~~
+
+select coalesce(a22, a21) from babel_726_t1
+GO
+~~START~~
+uniqueidentifier
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: invalid input syntax for type uuid: "asdfgh")~~
+
+
+-- nvarchar and nchar
+select coalesce(a22, a23) from babel_726_t1
+GO
+~~START~~
+nvarchar
+asdfgh
+~~END~~
+
+select coalesce(a23, a22) from babel_726_t1
+GO
+~~START~~
+nvarchar
+a              
+~~END~~
+
+
+-- nchar and varchar
+select coalesce(a23, a24) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The string size for the given CHAR/NCHAR data is not defined. Please use an explicit CAST or CONVERT to CHAR(n)/NCHAR(n))~~
+
+select coalesce(a24, a23) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The string size for the given CHAR/NCHAR data is not defined. Please use an explicit CAST or CONVERT to CHAR(n)/NCHAR(n))~~
+
+
+-- varchar and char
+select coalesce(a24, a25) from babel_726_t1
+GO
+~~START~~
+varchar
+asdfg
+~~END~~
+
+select coalesce(a25, a24) from babel_726_t1
+GO
+~~START~~
+varchar
+b              
+~~END~~
+
+
+-- char and varbinary
+select coalesce(a25, a26) from babel_726_t1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: COALESCE could not convert type varbinary to bpchar)~~
```

Comment:
is this expected?

Reply:
We do not have any IMPLICIT CAST between varbinary and bpchar. That is the reason this is failing. IMO, This should be tracked as a separate issue.